src/platform/plugins/shared/dashboard/server/dashboard_saved_object/schema/v2/v2.ts

```diff
@@ -13,6 +13,29 @@ import {
   dashboardAttributesSchema as dashboardAttributesSchemaV1,
 } from '../v1';
 
+export const sectionSchema = schema.arrayOf(
+  schema.object({
+    id: schema.maybe(
+      schema.string({
+        meta: { description: 'The unique ID of the section.' },
+      })
+    ),
+    order: schema.number({
+      min: 1,
+      meta: {
+        description:
+          'The order that sections should be rendered in. These values should be unique, and the order `0` is reserved for the main dashhboard content.',
+      },
+    }),
+    title: schema.string({
+      meta: { description: 'The title of the section.' },
+    }),
+    collapsed: schema.boolean({
```

Comment:
Can we make `collapsed` optional? Then, the UI can decide whether it is collapsed or not if `collapsed` is undefined.

Reply:
See https://github.com/elastic/kibana/pull/215052#discussion_r2031939738